internal/elements.go

```diff
@@ -404,3 +392,18 @@ type Set struct {
 	XMLName xml.Name `xml:"DAV: set"`
 	Prop    Prop     `xml:"prop"`
 }
+
+// https://tools.ietf.org/html/rfc6578#section-6.1
+type SyncCollectionQuery struct {
+	XMLName   xml.Name `xml:"DAV: sync-collection"`
+	SyncToken string   `xml:"sync-token"`
+	Limit     *Limit   `xml:"limit,omitempty"`
+	SyncLevel string   `xml:"sync-level,omitempty"`
+	Prop      *Prop    `xml:"prop,omitempty"`
```

Comment:
This is mandatory, so we can drop `,omitempty` and remove the pointer (like in `Set` above).